apps/passport-server/src/services/telegramService.ts

```diff
@@ -353,6 +271,38 @@ export class TelegramService {
       `,
         { parse_mode: "HTML", reply_to_message_id: messageThreadId }
       );
+      const msg = await ctx.reply(`Loading tickets and events...`);
+      const events = await fetchLinkedPretixAndTelegramEvents(
+        this.context.dbPool
+      );
+      const eventsWithChats = await getEventsWithChats(
+        this.context.dbPool,
+        ctx,
+        events
+      );
+
+      const userId = ctx.from?.id;
+      if (!userId) throw new Error(`No user found. Try again...`);
```

Comment:
do we have error handling within a bot command? or will this cause the thread to crash and the bot to not respond?

Reply:
We have top level error handling https://github.com/proofcarryingdata/zupass/blob/e7d494925c92610c73c86dca09143bf234b60cd3/apps/passport-server/src/services/telegramService.ts#L833 that seems to work. But also good to add here